Libraries/Microsoft.Teams.AI/Prompts/ChatPrompt/ChatPrompt.Send.cs

```diff
@@ -13,12 +13,17 @@ public async Task<IMessage> Send(IMessage message, CancellationToken cancellatio
         return await Send(message, null, null, cancellationToken);
     }
 
-    public async Task<ModelMessage<string>> Send(string text, OnStreamChunk? onChunk = null, CancellationToken cancellationToken = default)
+    public async Task<ModelMessage<string>> Send(string text, CancellationToken cancellationToken = default)
+    {
+        return await Send(text, null, null, cancellationToken);
+    }
+
+    public async Task<ModelMessage<string>> Send(string text, OnStreamChunk? onChunk, CancellationToken cancellationToken = default)
     {
         return await Send(text, null, onChunk, cancellationToken);
     }
 
-    public Task<ModelMessage<string>> Send(string text, IChatPrompt<TOptions>.RequestOptions? options = null, OnStreamChunk? onChunk = null, CancellationToken cancellationToken = default)
+    public Task<ModelMessage<string>> Send(string text, IChatPrompt<TOptions>.RequestOptions? options, OnStreamChunk? onChunk = null, CancellationToken cancellationToken = default)
```

Comment:
without these changes, we always got a "ambiguous function" if you tried to do Send("text") since it couldn't infer which one of these it should call